contracts/AirdropCampaignData.sol

```diff
@@ -0,0 +1,130 @@
+
+        airdrops[_airdropCampaignAddress].hashArray.push(_merkleRootHash);
+        emit MerkleRootHashAdded(_airdropCampaignAddress, _merkleRootHash);
+    }
+
+    /// @notice Adds array of daily merkle root hashes for multiple airdropCampaign contracts
+    /// @param _airdropCampaignAddressArray - array of addresses of multiple airdropCampaign contracts whose current participants are in the daily merkleRootHash
+    /// @param _merkleRootHashArray - array of merkle root hashes of daily participants of multiple airdropCampaigns
+    function batchAddDailyMerkleRootHash(address[] memory _airdropCampaignAddressArray, bytes32[] memory _merkleRootHashArray)
+        external
+        onlyAdmin
+    {
+        uint256 airdropHashArrayLength = _merkleRootHashArray.length;
+        if (airdropHashArrayLength > batchHashArrayLimit) revert ArrayTooLong();
+        if (airdropHashArrayLength != _airdropCampaignAddressArray.length) revert UnequalArrays();
+
+        for (uint256 i; i < airdropHashArrayLength; i++) {
+            address currentCampaignAddress = _airdropCampaignAddressArray[i];
+            bytes32 currentMerkleRootHash = _merkleRootHashArray[i];
+
+            if (airdrops[currentCampaignAddress].chain == Chains.Zero) revert ChainDataNotSet();
+            if (airdrops[currentCampaignAddress].airdropFinished == true) revert AirdropHasFinished();
+
+            airdrops[currentCampaignAddress].hashArray.push(currentMerkleRootHash);
+            emit MerkleRootHashAdded(currentCampaignAddress, currentMerkleRootHash);
+        }
+    }
+
+    /// @notice Adds string "ETH" or "POL" to mapping depending on which network the airdropCampaign contract is on
+    /// @param _airdropCampaignAddress - address of airdropCampaign contract
+    /// @param _airdropChain - string representing blockchain Chain
+    function addAirdropCampaignChain(address _airdropCampaignAddress, Chains _airdropChain) external onlyAdmin {
+        if (_airdropChain == Chains.Zero) revert ChainDataNotSet();
+        if (airdrops[_airdropCampaignAddress].chain != Chains.Zero) revert ChainAlreadySet();
+        if (airdrops[_airdropCampaignAddress].airdropFinished == true) revert AirdropHasFinished();
+
+        airdrops[_airdropCampaignAddress].chain = _airdropChain;
+        emit ChainAdded(_airdropCampaignAddress, _airdropChain);
+    }
+
+    /// @notice Adds array of Chain info: "Eth" or "Pol" to mapping depending on which Chain the airdropCampaign contracts are on
+    /// @param _airdropCampaignAddressArray - address of airdropCampaign contract
+    /// @param _airdropChainArray - array of uints representing blockchain chain
+    function batchAddAirdropCampaignChain(address[] memory _airdropCampaignAddressArray, Chains[] memory _airdropChainArray)
+        external
+        onlyAdmin
+    {
+        uint256 airdropChainArrayLength = _airdropChainArray.length;
+        if (airdropChainArrayLength > batchChainArrayLimit) revert ArrayTooLong();
```

Comment:
next line covers this

Reply:
Same a n previous example above, will discuss with teammates if we should remove the check for array length

Reply:
Confirmed - removing the check.